src/pages/inventory/inventory.html

```diff
@@ -26,23 +26,23 @@
   </ion-segment>
 
   <ion-item>
-    <ion-label floating>Brand</ion-label>
+    <ion-label color="primary">Brand</ion-label>
```

Comment:
That makes sense, thanks for the explanation! I was just making sure you didn't miss something, but I'm glad to know about this.